docs/UserGuide.md

```diff
@@ -564,31 +568,39 @@ This will edit an expenditure at **index 1** to be **renamed** "Long-Sleeve" wit
 Figure 6.3.3.1. Example of edit expenditure command
 #### 6.3.4. Finding expenditures: `find`
 (Contributed by Chin Hui)
-You can use this command to **filter expenditures by name**.
+
+You can use this command to **quickly find your expenditures**.
+
+Use this command to filter through your expenditures by keywords.
+NUSave will display all expenditures whose title contains the entered keyword/key phrase.
 
    ```📕 Note: Expenditures will be filtered as long as they contain the search term. The filter is case-insensitive, using upper-case or lower-case will not make a difference in the search result.```
 
-> ⚠️ **You can only use this command if there are existing expenditures**.
+> ⚠️ **You should not use this command if budget page is empty**. 
```

Comment:
Don't think you should use bold here as the symbol is sufficient. It is also more consistent with the other commands.

Reply:
Good spot! I'll change it ASAP. 😄